fusedrug/eval/metrics/protein_sequences.py

```diff
@@ -152,3 +152,24 @@ def compare_strings(from_text: Any, to_text: Any, return_score: bool = False) ->
         return ans
 
     return counts
+
+
+def _naive_aar_no_indels(preds: List[str], target: List[str]) -> List[float]:
+    # evlauates the edit distance, normalized by the input length, between 2 strings of the same lengths (i.e., no insertions and deletion)
+    # incases where len(preds) != len(target) raises an exception
```

Comment:
Annoying comment, but I think we should follow the standard and use:

```python 
def _naive_aar_no_indels(preds: List[str], target: List[str]) -> List[float]:
    """evlauates the edit distance, normalized by the input length, between 2 strings of the same lengths (i.e., no insertions and deletion)
    incases where len(preds) != len(target) raises an exception
    """
```

Reply:
changed